plugin/github-url.vim

```diff
@@ -25,13 +25,13 @@ function! s:path()
   return systemlist("git ls-files --full-name " . @%)[0]
 endfunction
 
-function! s:lineAnchor() range
-  let line = "#L" . a:firstline
-  if a:firstline != a:lastline
-    if repo=~#"gitlab\.com"
-      let line = line . "-" . a:lastline
+function! s:lineAnchor(repo, first, last) range
```

Comment:
You can probably drop the `range` on the function now if it's not using `a:firstline` and `a:lastline`.

Reply:
Good call: 22639f3dfcf042faed2f5917940b0f136f1ead47